File: Real_Estate_Recommendation_Platform_Fullstack/Back-end/Property.Admin/Api.Properties.Admin/Models/PropertyInfo.cs
using Properties.Admin;

namespace Api.Properties.Admin.Models
{
    public class PropertyInfo : BaseObject
    {
        public string PropertyType { get; set; }

        public string PropertyRegNum { get; set; }
        public string Description { get; set; }
        public string AreaSqft { get; set; }
        public string FurnishedStatus { get; set; }
        public string NumberOfFloors { get; set; }
        public string PropertyAge { get; set; }
        public string Address { get; set; }
        public string Price { get; set; }
        public string City { get; set; }
        public string Location { get; set; }
        public Category CategoryType { get; set; }
        public PropertyStatus PropertyStatus { get; set; }
        public Featured Featured { get; set; }
        public string AgencyId { get; set; }
        public string AgencyName { get; set; }
        public string AgencyPhoneNumber { get; set; }
        public string AgencyAddress { get; set; }
        public List<Image> Images { get; set; }
    }
}

File: Real_Estate_Recommendation_Platform_Fullstack/Front-end/Admin Front-end/findmyhome_admin/src/models/propertyInfo.js
export default class propertyInfo
{
    constructor(propertyType, 
                propertyRegNum, 
                description,
                areaSqft,
                furnishedStatus,
                numberOfFloors,
                propertyAge,
                address, 
                price, 
                city, 
                location, 
                categoryType,
                propertyStatus,
                featured,
                agencyId,
                agencyName,
                agencyPhoneNumber,
                agencyAddress,
                images,
                )
    {
        this.propertyType = propertyType;
        this.propertyRegNum = propertyRegNum;
        this.description = description;
        this.areaSqft = areaSqft;
        this.furnishedStatus = furnishedStatus;
        this.numberOfFloors = numberOfFloors;
        this.propertyAge = propertyAge;
        this.address = address;
        this.price = price;
        this.city = city;
        this.location = location;
        this.categoryType = categoryType;
        this.propertyStatus = propertyStatus;
        this.featured = featured;
        this.agencyId = agencyId;
        this.agencyName = agencyName;
        this.agencyPhoneNumber = agencyPhoneNumber;
        this.agencyAddress = agencyAddress;
        this.images = images;
    }

}